https://github.com/termux/termux-packages/issues/18792
https://bugzilla.mozilla.org/show_bug.cgi?id=1637465

This should be a temporary measure. Look into enable building the same
feature set as desktop.

--- a/modules/libpref/init/all.js
+++ b/modules/libpref/init/all.js
@@ -3711,7 +3711,7 @@
 
 pref("webextensions.storage.sync.enabled", true);
 // Should we use the old kinto-based implementation of storage.sync? To be removed in bug 1637465.
-pref("webextensions.storage.sync.kinto", false);
+pref("webextensions.storage.sync.kinto", true);
 // Server used by the old kinto-based implementation of storage.sync.
 pref("webextensions.storage.sync.serverURL", "https://webextensions.settings.services.mozilla.com/v1");
 
